gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -238,19 +257,17 @@ def compute_hard_filters(cov_threshold: int = 15, include_sex_filter: bool = Tru
     # These were determined by visual inspection of the metrics
     bi_allelic_qc_struct = get_sample_qc('bi_allelic').ht()[ht.key]
     hard_filters['bad_qc_metrics'] = (
-            (bi_allelic_qc_struct.sample_qc.n_snp > 3.75e6) |
-            (bi_allelic_qc_struct.sample_qc.n_snp < 2.4e6) |
-            (bi_allelic_qc_struct.sample_qc.n_singleton > 1e5) |
-            (bi_allelic_qc_struct.sample_qc.r_het_hom_var > 3.3)
+            (bi_allelic_qc_struct.sample_qc.n_snp > max_n_snp) |
+            (bi_allelic_qc_struct.sample_qc.n_snp < min_n_snp) |
+            (bi_allelic_qc_struct.sample_qc.n_singleton > max_n_singleton) |
+            (bi_allelic_qc_struct.sample_qc.r_het_hom_var > max_r_het_hom_var)
     )
 
     # Remove samples that fail picard metric thresholds, percents are not divided by 100, e.g. 5% == 5.00, %5 != 0.05
```

Comment:
I think this line can be removed since it is now noted in the docstring but I don't have a strong opinion on it.